spec/models/log_entry_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper' 
+
+describe LogEntry do 
+  describe 'entry_time' do 
+    before do 
+      Timecop.freeze(Time.local(2017, 03, 16, 10, 30, 0))
```

Comment:
Since we only have one test here, it might be nicer to use the `Timecop.freeze ... do` block syntax, so that we don't need `before` and `after` blocks.